search/tweets_engine.go

```diff
@@ -0,0 +1,89 @@
+package search
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"gopkg.in/olivere/elastic.v6"
+
+	"github.com/Bnei-Baruch/archive-backend/consts"
+	"github.com/pkg/errors"
+)
+
+func (e *ESEngine) SearchTweets(query Query, sortBy string, from int, size int, preference string) (map[string][]*elastic.SearchResult, error) {
+	tweetsByLang := make(map[string][]*elastic.SearchResult)
+	mssTweets := e.esc.MultiSearch()
+	mssTweets.Add(NewResultsSearchRequests(
+		SearchRequestOptions{
+			resultTypes:      []string{consts.ES_RESULT_TYPE_TWEETS},
+			index:            "",
+			query:            query,
+			sortBy:           sortBy,
+			from:             0,
+			size:             from + consts.TWEETS_SEARCH_COUNT,
```

Comment:
I think should always be from 0. Why from |from|?

Reply:
changed to

from:             from,
size:             consts.TWEETS_SEARCH_COUNT,

Reply:
fixed